packages/flame/test/sprite_animation_test.dart

```diff
@@ -41,6 +41,36 @@ void main() {
       );
     });
 
+    test('onStart called for single-frame animation', () {
+      var counter = 0;
+      final sprite = MockSprite();
+      final animation =
+          SpriteAnimation.spriteList([sprite], stepTime: 1, loop: false)
+            ..onStart = () => counter++;
+      expect(counter, 0);
+      animation.update(0.5);
+      expect(counter, 0);
```

Comment:
```suggestion
      expect(counter, 1);
```
This should be 1 (if the code is correct)

Reply:
That would be correct.